ci/lint/04_install.sh

```diff
@@ -6,9 +6,22 @@
 
 export LC_ALL=C
 
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+  # update first to install required ruby dependency
+  travis_retry brew update
+  travis_retry brew install grep # gnu grep for --perl-regexp support
+  PATH="$(brew --prefix grep)/libexec/gnubin:$PATH"
+  travis_retry brew install shellcheck
+  travis_retry brew upgrade python
+  PATH="$(brew --prefix python)/bin:$PATH"
+  export PATH
+else
+  SHELLCHECK_VERSION=v0.6.0
+  curl -s "https://storage.googleapis.com/shellcheck/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" | tar --xz -xf - --directory /tmp/
```

Comment:
Probably a good idea to add a `travis_retry` here as well

Reply:
Added `travis_retry` and switched `-s` to `--silent` for clarity.

@MarcoFalke will plan on a follow-up converting the lint scripts

Reply:
Ah, looks like setup_env is already running (because the `before_install` step is inherited.

https://travis-ci.org/bitcoin/bitcoin/jobs/595662274#L125